src/libraries/System.Runtime.InteropServices.JavaScript/src/System.Runtime.InteropServices.JavaScript.csproj

```diff
@@ -85,6 +85,7 @@
     <Reference Include="System.Net.Primitives" />
     <Reference Include="System.Runtime" />
     <Reference Include="System.Runtime.InteropServices" />
+    <Reference Include="System.Runtime.Loader" />
```

Comment:
What is the download size impact of this new dependency ?

Reply:
I'll check that

Reply:
The `AssemblyLoadContext` lives in `System.Private.CoreLib` and there is no size difference (comparing sample on main and on PR)